web/src/components/table/use-cases/scores.tsx

```diff
@@ -218,14 +287,16 @@ export default function ScoresTable({
       cell: ({ row }) => {
         const value = row.getValue("comment") as ScoresTableRow["comment"];
         return (
-          value !== undefined && (
-            <IOTableCell data={value} singleLine={rowHeight === "s"} />
-          )
+          !!value && <IOTableCell data={value} singleLine={rowHeight === "s"} />
         );
       },
     },
   ];
 
+  const columns = rawColumns.filter(
+    (c) => !!c.id && !hiddenColumns.includes(c.id),
+  );
+
   const [columnVisibility, setColumnVisibility] =
     useColumnVisibility<ScoresTableRow>("scoresColumnVisibility", columns);
```

Comment:
if the columns are different between the different embeds, we might need to pass a table name for each of them in order to save the local storage state on different strorage keys